source/dub/dub.d

```diff
@@ -1824,6 +1828,20 @@ private struct SpecialDirs {
 	 */
 	NativePath userPackages;
 
+	/**
+	 * Location at which build/generation artifact will be written
+	 *
+	 * All build artifacts are stored under a single build cache,
+	 * which is usually located under `$HOME/.dub/cache/` on POSIX,
```

Comment:
If we want to follow https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html then ~/.cache/dub could be an alternative directory to place cache files.

Reply:
That will be for another issue.